wandb/integration/diffusers/resolvers/utils.py

```diff
@@ -0,0 +1,46 @@
+import inspect
+from typing import Any, Dict, List, Optional, Sequence
+
+import numpy as np
```

Comment:
are you guaranteed that numpy will be installed? if not you need add a check and let the user know that numpy is required. you could you use `util.get_module`

Reply:
also if it is only used for specific utilities maybe it makes sense to move it to the function's scope

Reply:
Keeping `numpy` and `torch` imports outside the scope of functions because they're also being used for type annotation in a few cases.

Reply:
see my other comment